### PR TITLE
#1 - Using the Sqlite library on Windows

### PR DESCRIPTION
**Pull request by Mil0w0**
With those new files and modifications, once you have installed sqlite3 on your machine, you should be able to run the main.c code with functions from the Sqlite library.

Please, install it on your machines and try it out. 

Review it afterwards.

We will close the PR when everyone have reviewed it.

Happy New Year's Eve.

### CODE REVIEW

File: docs/installing_sqlite.md [hunk 1]
```diff
@@ -9,11 +11,24 @@ https://www.tutorialspoint.com/sqlite/sqlite_installation.htm
  * Add this path in the $PATH environnment variable (Paramètres>Système>A propos> Paramètres avancés> Environnement de variables> PATH> Nouveau>OK)
  * open a shell and run "slqite3" (-> should display version of sqlite)
```
Comment (M0UTOO) [inline docs/installing_sqlite.md]: This should be "run "sqlite3""  but we can understand it

**Review by M0UTOO — APPROVED** [review]
LGTM I get the test.db file when compiling with gcc on windows !

File: docs/installing_sqlite.md (hunk 1 above)
Comment (Mil0w0) [reply docs/installing_sqlite.md]: done

**Review by MB-Parmentier — APPROVED** [review]
Everything is working fine.

**Pull request merged**